tests/test_select.py

```diff
@@ -39,13 +39,13 @@ def test_path():
     assert hash(path4) != hash(path2)
 
 
-def test_selector_get_params_uniqueness():
+def test_selector_parameters_uniqueness():
     top = MockBrickTop(
         [MockBrickBottom(name="bottom"), MockBrickBottom(name="bottom")],
         name="top")
 
     selector = Selector([top])
-    assert_raises(ValueError, selector.get_params)
+    assert_raises(ValueError, selector.parameters)
```

Comment:
You're right. I'll fix it tomorrow morning.
On Jun 17, 2015 10:01 PM, "Xavier Bouthillier" <notifications@github.com>
wrote:

> In tests/test_select.py
> <https://github.com/mila-udem/blocks/pull/725#discussion_r32693904>:
>
> >      top = MockBrickTop(
> >          [MockBrickBottom(name="bottom"), MockBrickBottom(name="bottom")],
> >          name="top")
> >
> >      selector = Selector([top])
> > -    assert_raises(ValueError, selector.get_params)
> > +    assert_raises(ValueError, selector.parameters)
>
> get_
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mila-udem/blocks/pull/725/files#r32693904>.
>